### ISSUE TITLE
Bene: Fix mobile nav id attributes to be unique values from main menu

### ISSUE THREAD

**Issue by juliaford** [thinkshout/bene#110]
## Overview
<img width="1138" alt="screen shot 2017-09-13 at 1 16 18 pm" src="https://user-images.githubusercontent.com/28023410/30398793-873764ca-9886-11e7-87e6-da3ec341949b.png">

The ability to hover and click on the pencil only works once and then it doesn't let you return to a page or click on any other pencil to edit anything else. 

This actually raises some other issues since those menus have duplicated id attributes as well. We should we be rendering the mobile <nav> items with different id and data-contextual-id attributes. This will fix the above contextual link issue. 

## This ticket is done when:
- [ ] item
- [ ] item

**Comment by tauno** [comment]
This looks specific to the contextual edit on the menus.

**Comment by ruscoe** [comment]
This is happening to contextual edit buttons on menus because both the primary and secondary menus are rendered twice on the page (once for regular display, once for mobile):

https://github.com/thinkshout/bene/blob/8.x-1.x/themes/bene_base/templates/page.html.twig#L6-L16

This results in two contextual link elements having identical `data-contextual-id` properties, and that confuses the JavaScript.

I might be able to patch the Contextual module to add some other identifier to `data-contextual-id` to work around this.

It took forever to figure this out, btw. JavaScript should be banned.

**Comment by tauno** [comment]
This actually raises some other issues since those menus have duplicated `id` attributes as well. @heypaxton should we be rendering the mobile `<nav>` items with different `id` and `data-contextual-id` attributes?

**Comment by ruscoe** [comment]
@tauno That would solve the issue and be semantically correct.

**Comment by heypaxton** [comment]
@tauno yes we should have different IDs on the page. None of them should repeat.

**Comment by unclegcb** [comment]
@komejo @heypaxton I'm throwing an estimate on here but if one of you wants to change it, that would be swell.
